### PR TITLE
List

### PR DESCRIPTION
**Pull request by SumitSaundadkar**
List component.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -763,6 +763,79 @@ <h1>Navbar</h1>
...
 
+<div class="container">
+  <h1 id="list-group" >List</h1>
+  <p>Here we created List Component.</p>
+  <p>
+    The list component.
+  </p>
+  <ul class="list">
+    <div class="list-heading">
+      <img src="	https://openui.netlify.app/images/remove.svg" alt="">
```
Comment (ritikaawasthy) [inline index.html:775]: commit spacing here

**Review by ritikaawasthy — APPROVED** [review]
looks good to me

**Review by TusharJade — APPROVED** [review]
Looks go to me, good to go👍

File: index.html (hunk 1 above)
Comment (SumitSaundadkar) [reply index.html:775]: thanks for the review.

**Pull request merged**